src/functions/slack_function_handler_type_test.ts

```diff
@@ -37,10 +37,14 @@ Deno.test("SlackFunctionHandler with inputs and outputs", () => {
       },
     };
   };
-  const { createContext } = SlackFunctionTester("test");
+  const { createContext } = SlackFunctionTester(TestFn);
   const inputs = { in: "test" };
   const result = handler(createContext({ inputs }));
-  assertEquals(result.outputs?.out, inputs.in);
+
+  const stringTest = SlackFunctionTester("test");
+  const stringResult = handler(stringTest.createContext({ inputs }));
+
+  assertEquals(result.outputs?.out, inputs.in, stringResult.outputs?.out);
```

Comment:
If a developer uses `SlackFunctionHandler` we expect them to pass that same definition to `SlackFunctionTester`, but just wanted to add a quick sanity test to make sure that just in case they don't, it still works.